homeassistant/components/micropel/helper/crypto.py

```diff
@@ -0,0 +1,77 @@
+"""Cryptography for communicators."""
+
+
+class Crypto:
```

Comment:
To keep the complexity of the API away from HA, maybe move this into the PyPi package that is responsible for communication.

Reply:
Yes, it sound good. I will do it.